core/catalog/loaders/walmart_us.py

```
@@ -0,0 +1,122 @@
+
+This loader reads raw CSV snapshots and produces canonical CatalogSnapshot.
+"""
+
+from __future__ import annotations
+
+import hashlib
+from pathlib import Path
+
+from core.catalog.loaders.base import read_csv_rows
+from core.catalog.normalize.common import normalize_currency, normalize_unit, parse_decimal
+from core.catalog.provider import (
```

Flake8 (7.3.0)

[error] 'core.catalog.provider.CatalogLoader' imported but unused

(F401)